src/Backend/Modules/MiniBlog/Engine/Model.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+namespace Backend\Modules\MiniBlog\Engine;
+
+use Backend\Core\Engine\Model as BackendModel;
+use Backend\Core\Engine\Language as BL;
+
+/**
+ * This action will add a post to the blog module.
+ *
+ * @author Stef Bastiaansen <stef.bastiaansen@netlash.com>
+ * @author Jelmer Snoeck <jelmer.snoeck@netlash.com>
+ * @author Lander Vanderstraeten <lander.vanderstraeten@wijs.be>
+ */
+class Model
+{
+    const QRY_DATAGRID_BROWSE =
+        'SELECT i.id, i.title, UNIX_TIMESTAMP(i.created) AS created, i.user_id
+         FROM mini_blog AS i
+         WHERE i.publish = ? AND i.language = ?';
+
+    /**
+     * Deletes one or more items
+     *
+     * @param int $id
+     */
+    public static function delete($id)
+    {
+        $id = (int)$id;
+        $db = BackendModel::getContainer()->get('database');
```

Comment:
You can just use `BackendModel::get('database')`, leaving the getContainer() call away.